lessons/module-1/introduction-to-testing-javascript.md

```diff
@@ -28,25 +27,9 @@ Even worse: we can end up in a situation where changing code in one place causes
 
 Constantly poking at our code is tedious and it's not the best use of our time. We're better off writing some code that tests our other code for us. This sounds almost like a joke about programmers, but it's actually a really efficient way to maintain quality in our code base.
 
-## What are some of the different kinds of tests?
+## Unit Tests
 
-We test our code at a number of different levels:
-
-#### Unit Tests
-- Unit tests test one function or one object in isolation to make sure that it behaves the way we were expecting it to behave.  
-
-#### Integration Tests  
```

Comment:
I would maybe leave this section in. Since they will encounter these when working through js foundations.

Reply:
just the integration tests section